pkg/helpers/describe/describer.go

```diff
@@ -544,6 +546,33 @@ func describeBuildTriggers(triggers []buildv1.BuildTriggerPolicy, name, namespac
 	}
 }
 
+// describeBuildVolumes returns the description of a slice of build volumes
+func describeBuildVolumes(w *tabwriter.Writer, volumes []buildv1.BuildVolume) {
+	if len(volumes) == 0 {
+		formatString(w, "Volumes", "<none>")
+		return
+	}
+	formatString(w, "Volumes", " ")
+	fmt.Fprint(w, "\tName\tSource Type\tSource\tMounts\n")
+	for _, v := range volumes {
+		var sourceName string
+		switch v.Source.Type {
+		case buildv1.BuildVolumeSourceTypeSecret:
+			sourceName = v.Source.Secret.SecretName
+		case buildv1.BuildVolumeSourceTypeConfigMap:
+			sourceName = v.Source.ConfigMap.Name
+		default:
+			sourceName = "InvalidSourceType"
```

Comment:
Nit: `<InvalidSourceType: %q>` or just `<InvalidSourceType>` to make it explicitly differ from other output.